README.md

```diff
@@ -35,29 +35,10 @@ Alternatively, follow these steps to train it yourself.
 ### Prepare the Corpus
 
 1. Download the [Europarl Spanish-English parallel corpus](https://www.statmt.org/europarl/v7/es-en.tgz).
-2. Remove the sentences that don't form a pair (the sentences that correspond with an empty line).
-3. Remove sentences of length 1.
```

Comment:
From above, not this line (I can't reference the proper line...), can you please change `conda env update` for `conda env create`?